spec/controllers/agents/motifs_controller_spec.rb

```diff
@@ -73,21 +73,10 @@
         build(:motif, name: old_motif.name).attributes
       end
 
-      it "creates a new Motif" do
+      it "creates a new motif" do
         expect do
           post :create, params: { organisation_id: organisation_id, motif: valid_attributes }
-        end.to change(Motif, :count).by(0)
-      end
-
-      it "reactivate the motif" do
-        post :create, params: { organisation_id: organisation_id, motif: valid_attributes }
-        old_motif.reload
-        expect(old_motif.deleted_at).to be_nil
-      end
-
-      it "redirects to the created motif" do
-        post :create, params: { organisation_id: organisation_id, motif: valid_attributes }
-        expect(response).to redirect_to(organisation_motifs_path(organisation_id))
+        end.to change(Motif, :count).by(1)
```

Comment:
Juste une remarque: ca serait pas plus clean de le mettre dans une subject {} ? pour avoir un  expect { subject }.to ?

Reply:
oui tu as raison je fais ca